Library/Formula/git-cmd-hub.rb

```diff
@@ -0,0 +1,28 @@
+class GitCmdHub < Formula
+  desc "Interact with GitHub's v3 REST API using the git hub subcommand"
+  homepage "https://github.com/ingydotnet/git-hub"
+  url "https://github.com/ingydotnet/git-hub/archive/0.1.5.tar.gz"
+  sha256 "c057c2c56fa84a389105f935f63a72e5fa3f9fb9412a273d87b403b525ee2225"
+
+  def install
+    inreplace "Makefile" do |f|
+      # Prevent the makefile from installing outside of the prefix
+      f.gsub! "$(shell git --exec-path | sed 's/.*://')", "$(PREFIX)/bin"
+    end
+
+    inreplace "lib/git-hub" do |f|
+      # Fix the hardcoded directory to match Homebrew's structure
+      f.gsub! "SELFDIR=\"$(cd -P `dirname $BASH_SOURCE` && pwd -P)\"", "SELFDIR=#{bin}"
+    end
```

Comment:
Can you please submit these patches to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!